hathor/merged_mining/coordinator.py

```diff
@@ -1100,20 +1167,27 @@ class MergedMiningCoordinator:
     """
 
     BITCOIN_UPDATE_INTERVAL = 10.0
+    DIGIBYTE_UPDATE_INTERVAL = 3.0
     # very arbitrary, max times since last submit that we consider sending a new winning share
-    BITCOIN_MAX_FUTURE_SUBMIT_SECONDS = 30.0
+    MAX_FUTURE_SUBMIT_SECONDS = {
```

Comment:
Instead of `if` statements, isn't it easier to create a class `ParentSpecificMethods`? I don't know if it is easy to generalize but I feel it is better (and easier to add other parent chains in the future).

Reply:
I agree, but I used structures like `Dict[ParentChain, X]` precisely to avoid `if`s.

Reply:
I've seen `if` statements along the code. I understand that we can refactor it to generalize when we add the next parent chain. That's the idea, right?

Reply:
Yes. And IMO we would leave this for the next parent chain integration.